dockerfiles/README.md

```diff
@@ -21,6 +21,8 @@ To build vllm image do the following
    1. `git clone https://github.com/Kipok/OpenRLHF`
    2. checkout tag/commit
    3. run `pip install -e .`
+7. install flask with `pip3 install --ignore-installed flask` (needed for NeMo-Aligner GRPO)
+8. git clone NeMo-Aligner into /opt and run `pip3 install -e /opt/NeMo-Aligner/tensor_comms`
```

Comment:
`tensor_comms` ??

Reply:
yes, why is that concerning?

Reply:
does that path even exist? I don't see it in NeMo-Aligner repo

Reply:
it's using a custom version that's mentioned below. Let me add a reference here as well